lab-adam/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "mongodb-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "start": "DEBUG='city*' ./node_modules/nodemon/bin/nodemon.js server.js",
```

Comment:
Some of your files are using 'hospital*' with debug and thus you'll never see the debug statements in these pages.